src/cron/personalizedDigest.ts

```diff
@@ -1,11 +1,22 @@
 import fastq from 'fastq';
-import { notifyGeneratePersonalizedDigest } from '../common';
+import {
+  createEmailBatchId,
+  notifyGeneratePersonalizedDigest,
+} from '../common';
 import { UserPersonalizedDigest } from '../entity/UserPersonalizedDigest';
 import { Cron } from './cron';
 
 const cron: Cron = {
   name: 'personalized-digest',
   handler: async (con, logger) => {
+    const emailBatchId = await createEmailBatchId();
```

Comment:
Maybe I'm overlooking this, but where are you using the batchID besides the logs?
It should send through to sendgrid

Reply:
Yeah, still work in progress. Today I am going to test that sendgrid call works.

Reply:
Implemeneted here 7d286c82cbf2bd0b0fac6cbf53782c6391ec17e5